src/ext/java/org/opentripplanner/ext/transmodelapi/model/plan/ItineraryFiltersInputType.java

```diff
@@ -52,12 +52,22 @@ public static GraphQLInputObjectType create(GqlUtil gqlUtil, ItineraryFilterPara
         .field(GraphQLInputObjectField
             .newInputObjectField()
             .type(Scalars.GraphQLFloat)
-            .name(GROUP_SIMILARITY_KEEP_N_ITINERARIES)
+            .name(GROUP_SIMILARITY_KEEP_THREE)
             .description(
-                "Reduce the number of itineraries to the requested number by reducing each "
-                    + "group of itineraries grouped by 68% similarity.")
-            .defaultValue(dft.groupSimilarityKeepNumOfItineraries)
+                "Reduce the number of itineraries in each group to to maximum 3 itineraries. "
+                + "The itineraries are grouped by similar legs (on board same journey). So, if "
+                + " 68% of the distance is traveled by similar legs, then two itineraries are "
+                + "in the same group. Default value is 68%, must be at least 50%.")
+            .defaultValue(dft.groupSimilarityKeepThree)
             .build())
+        .field(GraphQLInputObjectField
```

Comment:
Is there a way to mark an input field as deprecated?

Reply:
No, unfotunatly.

Reply:
With the new version of GraphQL API it is now possible. Thanks for noticing. I will change all fields/input values this apply to.